documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -103,126 +103,114 @@ include::modules/installing-mtv-operator.adoc[leveloffset=+2]
 
 include::modules/configuring-mtv-operator.adoc[leveloffset=+2]
 
-
-[id="migrating-vms-web-console_{context}"]
+[id="migrating-vms-ocp-web-console_{context}"]
 == Migrating virtual machines by using the {ocp} web console
 
-You can migrate virtual machines (VMs) by using the {ocp} web console to:
-
-. xref:adding-source-providers[Create source providers]
-. xref:adding-destination-providers[Create destination providers]
-. xref:creating-migration-plans-ui[Create migration plans]
-
-[IMPORTANT]
-====
-You must ensure that all xref:prerequisites_{context}[prerequisites] are met.
-
-VMware only: You must have the minimal set of xref:vmware-privileges_{context}[VMware privileges].
-
-VMware only: Creating a xref:creating-vddk-image_{context}[VMware Virtual Disk Development Kit (VDDK)] image will increase migration speed.
-====
+Use the {project-short} user interface to migrate virtual machines (VMs). It is located in the *Virtualization* section of the {ocp} web console.
 
 include::modules/mtv-ui.adoc[leveloffset=+2]
-include::modules/mtv-overview-page.adoc[leveloffset=+2]
-include::modules/mtv-settings.adoc[leveloffset=+2]
-
-[id="adding-providers"]
-=== Adding providers
+include::modules/mtv-overview-page.adoc[leveloffset=+3]
+include::modules/mtv-settings.adoc[leveloffset=+3]
 
-You can add source providers and destination providers for a virtual machine migration by using the {ocp} web console.
+[id="migrating-vms-mtv-ui_{context}"]
+=== Migrating virtual machines using the MTV user interface
 
-[id="adding-source-providers"]
-==== Adding source providers
-
-You can use {project-short} to migrate VMs from the following source providers:
+Use the {project-short} user interface to migrate VMs from the following providers:
 
 * VMware vSphere
-* {rhv-full}
+* {rhv-full} ({rhv-short})
 * {osp}
 * Open Virtual Appliances (OVAs) that were created by VMware vSphere
-* {virt}
-
-You can add a source provider by using the {ocp} web console.
+* Remote {virt} clusters
```

Comment:
AFAIK, migration can also be from a local ocp provider (to a remote one).
So a source OCP provider can be either local or remote. 
@mnecas please confirm